docs/using.md

```diff
@@ -494,6 +511,74 @@ spec:
           ${inputs.resources.testCluster.Name} apply -f /workspace/service.yaml'
 ```
 
+### Storage Resource
+
+Storage resource represents a blob storage, that contains either an object or directory.
+Adding the storage resource as an input to a task will download the blob and allow the task to perform the required actions on the contents of the blob. Blob storage type "Google Cloud Storage" is supported as of now.
+
+#### GCS Storage Resource
+
+GCS Storage resource points to "Google Cloud Storage" blob.
+
+To create a GCS type of storage resource using the `PipelineResource` CRD:
+
+```yaml
+apiVersion: pipeline.knative.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: wizzbang-storage
+  namespace: default
+spec:
+  type: storage
+  params:
+    - name: type
+      value: gcs
+    - name: location
+      value: gs://some-bucket
+```
+
+Params that can be added are the following:
+
+1. `location`: represents the location of the blobstorage.
+1. `type`: represents the type of blob storage. Currently there is implementation
+   for only GCS.
+1. `dir`: represents whether the blob storage is a director or not.
+
+To use GCS private buckets, [service accounts](https://cloud.google.com/compute/docs/access/service-accounts
+) could be configured to access GCS bucket for `read` and `write` authorities. Download the service account keys to create a secret.
```

Comment:
i think there's an extra newline here that's interfering with `service accounts` being a link?

Reply:
I'm not sure what "`read` and `write` authorities" means - is this a GCS term? Or is this "or `read` and `write` permissions"?

Reply:
Permissions. I will correct it